advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,586 @@
+---
+title: 'Multiple Repositories'
+description: "Deeper look at the multiple repositories support feature"
+---
+
+The multiple repositories support was added in the [v2.33](https://github.com/pgbackrest/pgbackrest/releases/tag/release%2F2.33) release. This feature allow, for example, a local repository with minimal retention for fast restores and a remote repository with a longer retention for redundancy and access across the enterprise.
```

Comment:
Rephrase 'This feature allow, ...'
To
'One example where the Multiple Repositories could be used, would be for situations where a local repository would have minimal retention (for fast restores), and a remote repository would a longer retention (for redundancy and access across the enterprise).

Reply:
Fixed in 4c558e2